### PR TITLE
Feature/public distribution

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: MobileCenterDistribute/MobileCenterDistribute/Internals/Sender/MSDistributeSender.h [hunk 1]
```diff
@@ -20,15 +20,18 @@ static NSString *const kMSHeaderUpdateApiToken = @"x-api-token";
  *
  * @param baseUrl Base url.
  * @param appSecret A unique and secret key used to identify the application.
- * @param updateToken The update token stored in keychain.
+ * @param updateToken The update token stored in keychain. This parameter is optional and the update will be considered
+ * as public distribution if it is nil.
+ * @param distributionGroupId The distribution group Id stored in keychain.
```
[the remark this answers is not in the document]
Comment (jaelim-ms) [reply MobileCenterDistribute/MobileCenterDistribute/Internals/Sender/MSDistributeSender.h]: Yeah.. That's not a sensitive data. Don't need to keep that in keychain.

**Comment by jaelim-ms** [comment]
Nice catch! I fixed the typo.

File: MobileCenterDistribute/MobileCenterDistribute/Internals/Sender/MSDistributeSender.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (clpolet) [reply MobileCenterDistribute/MobileCenterDistribute/Internals/Sender/MSDistributeSender.h]: Yes, you should be able to update the `@param distributionGroupId The distribution group Id stored in keychain.` line.

[the remark this answers is not in the document]
Comment (jaelim-ms) [reply MobileCenterDistribute/MobileCenterDistribute/Internals/Sender/MSDistributeSender.h]: Oops. I didn't get your point. I updated the doc.

**Pull request merged**